generators/php/sdk/src/endpoint/request/WrappedEndpointRequest.ts

```diff
@@ -176,9 +189,229 @@ export class WrappedEndpointRequest extends EndpointRequest {
+                }
+            ];
+
+            if (contentType) {
+                toMultipartArgs.push({
+                    name: "contentType",
+                    assignment: php.codeblock("TODO")
+                });
+            }
+
+            if (isOptional) {
+                writer.controlFlow("if", php.codeblock(`${requestVarName}->${propWireName} != null`));
+            }
+            if (isSequence) {
+                writer.controlFlow("foreach", php.codeblock(`${requestVarName}->${propWireName} as $file`));
+            }
+            writer.writeNodeStatement(
+                php.invokeMethod({
+                    method: "addPart",
+                    on: php.codeblock(formVarName),
+                    multiline: true,
+                    arguments_: [
+                        php.invokeMethod({
+                            method: "toMultipartFormDataPart",
+                            on: php.codeblock(isSequence ? "$file" : `${requestVarName}->${propWireName}`),
+                            arguments_: toMultipartArgs
+                        })
+                    ]
+                })
+            );
+            if (isSequence) {
+                writer.endControlFlow();
+            }
+            if (isOptional) {
+                writer.endControlFlow();
+            }
+        });
+    }
+
+    private genDataPartFromBodyProperty({
+        bodyProp,
+        requestVarName,
+        formVarName
+    }: {
+        bodyProp: FernIr.InlinedRequestBodyProperty;
+        requestVarName: string;
+        formVarName: string;
+    }): php.CodeBlock | undefined {
+        // TODO: Define how maps should be encoded in multi-part form uploads (prob wrap in json_encode(...))
+        if (this.context.isMap(bodyProp.valueType)) {
+            return undefined;
+        }
+
+        const propName = this.context.getPropertyName(bodyProp.name.name);
+        const isOptional = this.context.isOptional(bodyProp.valueType);
+        const isSequence = this.context.isSequence(bodyProp.valueType);
+
+        let valueBlock = isSequence ? "$item" : `${requestVarName}->${propName}`;
+        if (this.context.isEnum(bodyProp.valueType)) {
+            valueBlock += "->value";
+        } else if (this.context.isObject(bodyProp.valueType)) {
+            valueBlock += "->toJson()";
+        } else if (this.context.isUnknown(bodyProp.valueType)) {
+            valueBlock = `json_encode(${valueBlock})`;
+        }
+
+        const addArgs: Arguments = [
+            {
+                name: "name",
+                assignment: php.codeblock(`'${bodyProp.name.wireValue}'`)
+            },
+            {
+                name: "value",
+                assignment: php.codeblock(valueBlock)
+            }
+        ];
+
+        // if (bodyProp.contentType) {
+        //     addArgs.push({
+        //         name: "contentType",
+        //         assignment: php.codeblock(contentType)
+        //     });
+        // }
```

Comment:
Uncomment this once the ir-sdk has been updated to latest. current version doesn't have content type fields.